tools/postgres/schema.sql

```diff
@@ -685,9 +686,9 @@ CREATE TABLE webknossos.analyticsEvents(
   created TIMESTAMPTZ NOT NULL DEFAULT NOW(),
   sessionId BIGINT NOT NULL,
   eventType VARCHAR(512) NOT NULL,
-  eventProperties JSONB NOT NULL,
+  eventProperties JSONB NOT NULL, -- TODO migrate user json from old to new organization id
```

Comment:
@MichaelBuessemeyer is this TODO still open?

Reply:
nope, should be resolved